Frameworks/UIKit/UIPopoverPresentationController.mm

```diff
@@ -79,7 +71,6 @@ @implementation UIPopoverPresentationController {
     StrongId<UIBarButtonItem*> _barButtonItem;
     StrongId<UIView*> _sourceView;
     CGRect _sourceRect;
-    StrongId<dispatch_block_t> _dismissCompletion;
```

Comment:
No need for a private dismissCompletion API anymore (for the one user UIViewController.mm). This was used for the unlinking of the modal from the parent which is now handled in WYPopoverController (doing the unlinking after didDismissPopover is way too late).